CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.1.0
```

Comment:
Would this be 4.0.0?

Reply:
Oh! I think this disappears from the diff if we merge in the base branch?

Reply:
This got merged in from master. I'll add some more release notes here for these removals